xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/historyinline.vm

```diff
@@ -158,10 +158,12 @@ $xwiki.ssfx.use('uicomponents/pagination/pagination.css', true)##
         #if ($xwiki.hasMinorEdit())
           #if ("$!request.showminor" != 'true')
             <input class="btn btn-default" type="submit" name="viewminorversions" value="$services.localization.render('core.viewers.history.showMinorEdits')"
-              onclick="document.forms.${formname}.action='$tdoc.getURL('view', "viewer=history&amp;showminor=true&amp;$docvariant")'; if (document.forms.${formname}.onsubmit) document.forms.${formname}.onsubmit();" />
+              onclick="document.forms.${formname}.action='$tdoc.getURL('view', "$docvariant")'; if (document.forms.${formname}.onsubmit) document.forms.${formname}.onsubmit();" />
           #else
+          <input type="hidden" name="showminor" value="true" />
+          <input type="hidden" name="viewer" value="history" />
```

Comment:
bad indentation BTW

Reply:
Fixed that! let me know if there are any other errors, and I'll make those changes too and push them :)